### PR TITLE
Renamed gpscp_help to gpsync_help

### PR DESCRIPTION
**Pull request by Annu149**
Replaced scp with rsync and all gpscp references with gpsync in gpscp_help doc file and renamed it to gpsync_help as the next step of PR https://github.com/greenplum-db/gpdb/pull/14145

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/doc/gpsync_help [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 *****************************************************
 DESCRIPTION
 *****************************************************
 
 
-The gpscp utility allows you to copy one or more files from the 
+The gpsync utility allows you to copy one or more files from the 
 specified hosts to other specified hosts in one command using 
-SCP (secure copy). For example, you can copy a file from the 
+RSYNC (secure copy). For example, you can copy a file from the
```
Comment (shrakesh) [inline gpMgmt/doc/gpsync_help:24]: I am wondering do we need to write the full form of rsync  in place of secure copy ?

**Review by shrakesh — COMMENTED** [review]
there are lot of places where the term "RSYNC session" is mentioned. can we make it general instead of underline api specific. just a thought to discuss.  or may be we can avoid specifying RSync and can use generic statement instead.

File: gpMgmt/doc/gpsync_help [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 *****************************************************
 
 
-The gpscp utility allows you to copy one or more files from the 
+The gpsync utility allows you to copy one or more files from the 
 specified hosts to other specified hosts in one command using 
-SCP (secure copy). For example, you can copy a file from the 
-Greenplum Database coordinator host to all of the segment hosts at 
-the same time.
+remote sync (remote synchronization). For example, you can copy a file
```
Comment (shrakesh) [inline gpMgmt/doc/gpsync_help:24]: **remote sync (remote synchronization)**  can be replaced with **remote synchronization**

**Review by shrakesh — APPROVED** [review]
LGTM

File: gpMgmt/doc/gpsync_help [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 Copies files between multiple hosts at once.
 
 *****************************************************
 SYNOPSIS
 *****************************************************
 
-gpscp { -f <hostfile_gpssh> | -h <hostname> [-h <hostname> ...] } 
-[-J <character>] [-v] [[<user>@]<hostname>:]<file_to_copy> [...] 
+gpsync { -f <hostfile_gpssh> | -h <hostname> [-h <hostname> ...] } 
+[-a] [-J <character>] [-v] [[<user>@]<hostname>:]<file_to_copy> [...]
```
Comment (piyushc01) [inline gpMgmt/doc/gpsync_help:10]: Did we add `-a` as a new option?

Comment (Annu149) [reply gpMgmt/doc/gpsync_help:10]: yes

Comment (piyushc01) [reply gpMgmt/doc/gpsync_help:10]: Do we want to add description/help on the same?

Comment (Annu149) [reply gpMgmt/doc/gpsync_help:10]: sure, added description

**Review by piyushc01 — APPROVED** [review]

**Comment by kainwen** [comment]
Seems after this, make install print a message:

> /usr/bin/install: cannot stat 'gpscp_help': No such file or directory

**Comment by Annu149** [comment]
> Seems after this, make install print a message:
> 
> > /usr/bin/install: cannot stat 'gpscp_help': No such file or directory

Thanks for pointing it out, I will take a look

**Pull request merged**